### PR TITLE
tip on viewing node async output, see issue #1468

### LINKED ISSUE #1468: Some async output does not appear when using node repl in Calva with shadow-cljs
Minimal repro created with create-cljs-project. Using shadow-cljs
```
(defn print-with-delay
  [delay]
  (let [timer-id (js/setTimeout
                  #(println "delayed" delay "ms") delay)]
    timer-id))

(defn print-to-maxdelay
  [maxdelay]
  (doseq [delay (range 1 maxdelay)]
    (print-with-delay delay)))
```
Running (print-to-maxdelay 50) results in the following output in output.calva-repl window:
```
cljs꞉demo꞉> 
delayed 1 ms
delayed 2 ms
delayed 3 ms
delayed 4 ms
delayed 5 ms
nil
cljs꞉demo꞉> 
delayed 1 ms
delayed 2 ms
delayed 3 ms
delayed 4 ms
nil
```
on two successive runs. If the same code is compiled with shadow-cljs and run with node, however, all delays up to 49 ms are printed, as expected. The Calva repl is dropping all output after 4 or 5 ms. Complete minimal repro can be cloned from:
https://github.com/agoldhammer/minrepro

### PR DESCRIPTION
**Pull request by agoldhammer**
## What has Changed?

- Describes workaround for issue discussed in #1468.
- @pez suggests creating a Tips and Tricks section under Etcetera where this might fit.
-

Fixes #1468

## My Calva PR Checklist

I have:

- [x ] Read [How to Contribute](https://github.com/BetterThanTomorrow/calva/wiki/How-to-Contribute#before-sending-pull-requests).
- [x ] Directed this pull request at the `dev` branch. (Or have specific reasons to target some other branch.)
- [ ] Made sure I have changed the PR base branch, so that it is not `published`. (Sorry for the nagging.)
- [x ] - directed at "published" because this is a doc fix only
- [ ] Updated the `[Unreleased]` entry in `CHANGELOG.md`, linking the issue(s) that the PR is addressing.
- [x ] Figured if **anything** about the fix warrants tests on Mac/Linux/Windows/Remote/Whatever, and either tested it there if so, or mentioned it in the PR.
- [x ] Added to or updated docs in this branch, if appropriate

- [x ] Created the issue I am fixing/addressing, if it was not present.

Ping @pez, @bpringe

### CODE REVIEW

File: docs/site/async-out.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+description: Tips on using Calva to create Node scripts using shadow-cljs
+---
+
+# Viewing Async Output While Working On Node Projects with `shadow-cljs`
+
+When working on Node projects with `shadow-cljs` and Calva, async output does not always appear in the Calva output window. To work around this problem, follow these steps:
+
+1. In a terminal run `npx shadow-cljs -d cider/cider-nrepl:0.27.4 node-repl`
+2. Wait for the message `shadow-cljs - nREPL server started on port <some-port>`
+3. Issue the command **Calva: Connect to a running REPL server in your project**, `ctrl+alt+c ctrl+alt+c`. For project type select `shadow-cljs`, accepted the proposed `localhost:<some-port>`, and for `build` select 'node-repl`.
```
Comment (bpringe) [inline docs/site/async-out.md]:
```suggestion
3. Issue the command **Calva: Connect to a running REPL server in your project**, `ctrl+alt+c ctrl+alt+c`. For project type select `shadow-cljs`, accepted the proposed `localhost:<some-port>`, and for `build` select `node-repl`.
```

**Review by bpringe — CHANGES_REQUESTED** [review]
Thanks for adding this. Just a small change requested (typo).

**Comment by bpringe** [comment]
You can add this under Etcetera in the menu by using the `mkdocs.yml` file.

**Comment by agoldhammer** [comment]
> You can add this under Etcetera in the menu by using the `mkdocs.yml` file.

Thanks. Awaiting @pez instructions on what branch to target.

**Comment by PEZ** [comment]
@agoldhammer , directing it at `published` is fine.

**Comment by PEZ** [comment]
Thanks! Have you tested to build the docs site locally?

**Comment by agoldhammer** [comment]
Yes, works for me.

**Comment by PEZ** [comment]
🎉 Thanks! 🙏 ❤️

**Pull request merged**